collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderList.java

```diff
@@ -170,7 +144,7 @@ public void onClick(View v) {
         });
         mToggleButton.setOnLongClickListener(this);
 
-        Cursor c = mShowUnsent ? getUnsentCursor() : getAllCursor();
+        Cursor c = mShowUnsent ? new InstancesDao().getFinalizedInstancesCursor() : new InstancesDao().getAllCompletedInstancesCursor();
```

Comment:
What are the implications of these several `InstancesDao` object creations? I believe that as long as `InstancesDao` has no state this is ok but perhaps we should get in the habit of creating a variable when there's more than one call (as you did in other classes). This also improves readability a bit.